src/StatementBase.php

```diff
@@ -5,36 +5,36 @@ class StatementBase extends Element {
   protected $props = [
     'actor' => 'Locker\XApi\Actor',
     'verb' => 'Locker\XApi\Verb',
-    'object' => 'Locker\XApi\Object',
+    'statementobject' => 'Locker\XApi\StatementObject',
```

Comment:
I don't think this should have changed. There are other references to this here that I don't think should have changed.
```suggestion
    'object' => 'Locker\XApi\StatementObject',
```

Reply:
see #18